payment/__init__.py

```diff
@@ -0,0 +1,56 @@
+from flask import Flask, request, jsonify
+from .middleware import handle_errors
+from .models import Transaction
+from .utils import lock
+from . import blockchain
+
+
+app = Flask(__name__)
+
+
+@app.route('/wallets/', methods=['POST'])
+@handle_errors
+def create_wallet():
+    body = request.get_json()
+    wallet_address = body['wallet_address']
+
+    # wallet creation is idempotent - no locking needed
+    blockchain.create_wallet(wallet_address)
+
+    return jsonify(), 201
+
+
+@app.route('/orders/<order_id>', methods=['GET'])
+@handle_errors
+def get_order(order_id):
+    t = Transaction.get_by_order_id(order_id)
+    return jsonify(t.to_primitive())
+
+
+@app.route('/orders/', methods=['POST'])
+@handle_errors
+def pay():
```

Comment:
POST to `/orders` usually creates a new order

Reply:
It's in the process of creating - returing 202